src/components/SummaryTable/index.jsx

```diff
@@ -10,41 +11,56 @@ const SummaryTable = ({ content = [], header }) => {
     .map((name, i) => (name === compareName ? i : null))
     .exists()
     .first(); // TODO: replace with findIndex()
-return (
-  <table className='summary-table'>
-    {header && (
-    <thead>
-      <tr>
-        <th />
-        {header.map(item => <th className='column' key={item}>{item}</th>)}
-      </tr>
-    </thead>
+
+  return (
+    <table className="summary-table">
+      {header && (
+        <thead>
+          <tr>
+            <th />
+            {header.map(item => (
+              <th className="column" key={item}>
+                {item}
+              </th>
+            ))}
+          </tr>
+        </thead>
       )}
-    <tbody>
-      {content.map(({ dataPoints = [], statusColor, summary, title, uid }) => (
-        <tr key={uid}>
-          <td className='title-container'>
-            <StatusWidget statusColor={statusColor} title={title} />
-          </td>
-          {dataPoints.map((datum, columnIndex) => {
-              const className = columnIndex === compareColumn ? `status-${statusColor}` : '';
-              return (<td key={columnIndex} className={className}>{datum}</td>);
-            })}
-          <td>{summary}</td>
-        </tr>
-              ))}
-    </tbody>
-  </table>
+      <tbody>
+        {content.map(
+          ({ dataPoints = [], statusColor, summary, title, uid }) => (
+            <tr key={uid}>
+              <td className="title-container">
+                <StatusWidget statusColor={statusColor} title={title} />
+              </td>
+              {dataPoints.map((datum, columnIndex) => {
```

Comment:
Presumably the `eslint-disable react/no-array-index-key` is for this columnIndex being used as the key. Instead of disabling this rule, can you find an attribute on datum (like a name or id) that can be used instead?

Reply:
This helped. This has been fixed now.